grid/grid.css

```diff
@@ -0,0 +1,30 @@
+@import url("../styles.css");
+@import url("../cards/card.css");
+
+.grid--container{
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+    grid-row-gap : 1rem;
+}
+.child-grid-1{
```

Comment:
display, width and margin being repeated for each child-grid, you can optimize this by extracting them out in a common class

Reply:
Yeah, done that. Thanks a lot, Omkar for the feedback and code review.